routes/author/index.js

```diff
@@ -1,15 +1,14 @@
-"use strict";
+import Author from 'models/Author';
+import Booklist from 'models/Book';
+import _ from 'underscore';
+import Q from 'q';
 
-var Author = require('models/Author');
-var Booklist = require('models/Book');
-var _ = require('underscore');
-var Q = require('q');
-
-module.exports = function(req, res){
-  var isLogined = req.session.passport.user;
-  var encodeImgSrc = function(bookList) {
-    return bookList.map(function(book) {
-      var err, images;
+export default function(req, res) {
+  const isLogined = req.session.passport.user;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).